WebColock.Tests/IntegrationTest/IntegrationClockInOutControllerTests.cs

```diff
@@ -14,5 +15,14 @@ await server
                 .DoPost<object>($"api/ClockInOut/ClockIn/{userId}", new object());
             //todo usunąć <object>
         }
+        [Fact]
+        public async void ClockOut_Test()
+        {
+            var server = new ServerSut();
+            var userId = 1;
+
+            await server
+                .DoPost<object>($"api/ClockInOut/ClockOut/{userId}", new object());
```

Comment:
Domyślam się, że w metodzie DoPost jest jakieś sprawdzenie, czy wszystko poszło ok. Natomiast bardziej podoba mi się wzorzec 3A: "Arrange Act Assert".

W przyszłości możecie spróbować to przerobić.
Krótkie info tu: http://wiki.c2.com/?ArrangeActAssert

Reply:
W tym przypadku Assert jest wykonywany tak jak powiedziałeś, w metodzie DoPost, a tutaj nie ma potrzeby robić innych assertów.

Reply:
Ok. Warto jednak wiedzieć jak potem te testy pisać, żeby fajnie się je czytało. To AAA to jest pewien standard - więc warto się z nim oswoić ;-)